24_clustering.Rmd

```diff
@@ -17,9 +18,122 @@ features/samples that share pattern.  For example, with clustering, we
 can find group of samples that share similar community
 composition. There are multiple clustering algorithms available.
 
+## Introduction to bluster package
+
+*bluster* is a Bioconductor package specialized in clustering. It offers 
+multiple algorithms such as hierarchical clustering, DBSCAN, K-means, 
+amongst others. The first thing to do when using this package is to load 
+it. It's also possible to use `transformCounts` to transform the data 
+for the clustering.
+```{r load-bluster}
+# Load dependencies
+if (!require(bluster)) {
+  install.packages("bluster")
+  library(bluster)
+}
+
+# Optional: Apply transformation
```

Comment:
Remove "Optional:" or explain in what situations this should be done.

I prefer the first (removing the word "Optional")